internal/collector/patroni.go

```diff
@@ -69,23 +69,23 @@ func EnablePatroniLogging(ctx context.Context,
 					`set(instrumentation_scope.name, "patroni")`,
 
 					// https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/-/pkg/ottl/ottlfuncs#parsejson
-					`set(cache, ParseJSON(body["original"]))`,
+					`set(log.cache, ParseJSON(log.body["original"]))`,
```

Comment:
🤔 Do we still have to set `context=log`? If so, what a strange change they made.

Reply:
We do not. I'm not sure of the benefit of this change.

That said, is it expected for almost every emission to be `LogRecord #0`?

Reply:
That sounds like an index, maybe? The batch time is quite small by default, so one-at-a-time sounds plausible.